build.sbt

```diff
@@ -1,4 +1,4 @@
-import Dependencies._
+import Dependencies.{Libraries, _}
```

Comment:
why?

Reply:
I'm not sure how that happened. It has been rectified.